test/api/common.js

```diff
@@ -34,6 +38,18 @@ describe('Common API', function() {
         });
     });
 
+    describe('GET /api/exchanges', function() {
```

Comment:
I think this description and test helper could go to new: `test/api/exhanges.js`.